.github/workflows/workflow-build-and-deploy-scaleway.yml

```diff
@@ -56,8 +60,63 @@ on:
         required: true
 
 jobs:
-  build-and-deploy:
-    name: Build & Deploy to ${{ inputs.environment }}
+  monorepo-check:
```

Comment:
- Rename this to `generate-matrix`, like the other workflows.
- If there's a monorepo file, use that to check for changes
- If there isn't, create a matrix with only 1 entry: "empty" environment (i.e. empty suffix), and the root path.